app/Repositories/Eloquents/UserRepository.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Repositories\Eloquents;
+
+use Auth;
+use App\Models\User;
+use App\Repositories\Eloquents\BaseRepository;
+use App\Repositories\Interfaces\UserRepositoryInterface;
+
+class UserRepository extends BaseRepository implements UserRepositoryInterface
+{
+    public function getModel()
+    {
+        return User::class;
+    }
+
+    public function getAllAndSearch($perPage, $searchParams)
+    {
+        $query = $this->model->orderBy('created_at', 'desc');
+
+        if ($searchParams) {
+            $searchParams = $this->handleSearchParams($searchParams);
+
+            return $query->search($searchParams, $perPage);
+        } else {
+            return $query->paginate($perPage);
+        }
+    }
+
+    private function handleSearchParams($searchParams)
+    {
+        $searchKeys = ['name', 'email'];
+        foreach ($searchKeys as $searchKey) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/UserRepository.php</br>**Line**: 35</br>String "" does not require double quotes; use single quotes instead